scripts/hudson/narayana.bat

```diff
@@ -71,7 +71,7 @@ goto:eof
   git clone https://github.com/jbosstm/microprofile-lra.git || (call:comment_on_pull "MP LRA clone Failed %BUILD_URL%" & exit -1)
   if %ERRORLEVEL% NEQ 0 exit -1
   cd microprofile-lra
-  git checkout microprofile-lra
+  git checkout microprofile-lra-v2
```

Comment:
I guess the quickstart needs a PR with this too?

Reply:
There was no quickstart for LRA proxy just a test suite in master.

Reply:
Sure - I was meaning the branch that you are building: https://github.com/jbosstm/quickstart/blob/master/scripts/hudson/quickstart.sh#L28